store/src/java/com/zimbra/cs/html/owasp/OwaspHtmlSanitizer.java

```diff
@@ -123,6 +131,9 @@ public String cleanMalformedHtml(String str, boolean printBodyOnly) throws Unsup
             if (outStream == null || outStream.trim().isEmpty()) {
                 return str;
             }
+            if (outStream.contains("<\n/")) {
+                outStream = outStream.replaceAll("<\n/","</");
```

Comment:
String.replaceAll() takes regex as 1st argument, did you check behaviour with `\` in text? I suggest to use String.replace(), if there is no need of regex matching.
Also it would be great, if you can add unit test for this specific case.